src/box/errcode.h

```diff
@@ -115,286 +115,286 @@ struct errcode_record {
-	_(ER_UPDATE_INTEGER_OVERFLOW, 95,	"Integer overflow when performing '%c' operation on field %s") \
+	_(ER_CANT_UPDATE_PRIMARY_KEY, 94,	"Attempt to modify a tuple field which is part of primary index", "space", STRING) \
+	_(ER_UPDATE_INTEGER_OVERFLOW, 95,	"Integer overflow", "operation", STRING, "field", STRING) \
 	_(ER_GUEST_USER_PASSWORD, 96,		"Setting password for guest user has no effect") \
 	_(ER_TRANSACTION_CONFLICT, 97,		"Transaction has been aborted by conflict") \
-	_(ER_UNSUPPORTED_PRIV, 98,		"Unsupported %s privilege '%s'") \
-	_(ER_LOAD_FUNCTION, 99,			"Failed to dynamically load function '%s': %s") \
-	_(ER_FUNCTION_LANGUAGE, 100,		"Unsupported language '%s' specified for function '%s'") \
-	_(ER_RTREE_RECT, 101,			"RTree: %s must be an array with %u (point) or %u (rectangle/box) numeric coordinates") \
-	_(ER_PROC_C, 102,			"%s") \
-	_(ER_UNKNOWN_RTREE_INDEX_DISTANCE_TYPE, 103, "Unknown RTREE index distance type %s") \
-	_(ER_PROTOCOL, 104,			"%s") \
-	_(ER_UPSERT_UNIQUE_SECONDARY_KEY, 105,	"Space %s has a unique secondary index and does not support UPSERT") \
-	_(ER_WRONG_INDEX_RECORD, 106,		"Wrong record in _index space: got {%s}, expected {%s}") \
-	_(ER_WRONG_INDEX_PARTS, 107,		"Wrong index part %u: %s") \
-	_(ER_WRONG_INDEX_OPTIONS, 108,		"Wrong index options: %s") \
-	_(ER_WRONG_SCHEMA_VERSION, 109,		"Wrong schema version, current: %d, in request: %llu") \
-	_(ER_MEMTX_MAX_TUPLE_SIZE, 110,		"Failed to allocate %u bytes for tuple: tuple is too large. Check 'memtx_max_tuple_size' configuration option.") \
-	_(ER_WRONG_SPACE_OPTIONS, 111,		"Wrong space options: %s") \
-	_(ER_UNSUPPORTED_INDEX_FEATURE, 112,	"Index '%s' (%s) of space '%s' (%s) does not support %s") \
-	_(ER_VIEW_IS_RO, 113,			"View '%s' is read-only") \
+	_(ER_UNSUPPORTED_PRIV, 98,		"Unsupported privilege", "object_type", STRING, "privilege", STRING) \
+	_(ER_LOAD_FUNCTION, 99,			"Failed to dynamically load function", "function", STRING, "details", STRING) \
+	_(ER_FUNCTION_LANGUAGE, 100,		"Unsupported function language", "language", STRING, "function", STRING) \
+	_(ER_RTREE_RECT, 101,			"Coordinates number does not make a point or rectangle", "object", STRING, "dimension", UINT) \
+	_(ER_PROC_C, 102,			"Error in C function") \
+	/* ER_UNKNOWN_RTREE_INDEX_DISTANCE_TYPE, 103, Unused */ \
+	_(ER_PROTOCOL, 104,			"Protocol error", "details", STRING) \
+	/* ER_UPSERT_UNIQUE_SECONDARY_KEY, 105,	Unused */ \
+	_(ER_WRONG_INDEX_RECORD, 106,		"Wrong record in _index space", "actual", STRING, "expected", STRING) \
+	_(ER_WRONG_INDEX_PARTS, 107,		"Wrong index part", "partno", UINT, "details", STRING) \
+	_(ER_WRONG_INDEX_OPTIONS, 108,		"Wrong index options", "details", STRING) \
+	_(ER_WRONG_SCHEMA_VERSION, 109,		"Request schema version does not match database schema version", "actual", ULLONG, "expected", ULLONG) \
+	_(ER_MEMTX_MAX_TUPLE_SIZE, 110,		"Failed to allocate tuple: tuple is too large", "tuple_size", UINT) \
```

Comment:
"Tuple size is too big", "value", "limit". Also, let's use either "limit" (not "max") everywhere where a limit is exceeded.

Reply:
Done.

```diff
+       _(ER_MEMTX_MAX_TUPLE_SIZE, 110,         "Tuple size is too big", "value", ULLONG, "max", ULLONG) \
```

Also as discussed f2f changed "limit" to "max" in other places.

<details>
<summary>diff</summary>

```diff
diff --git a/src/box/errcode.h b/src/box/errcode.h
index afb8eec26..0313f9197 100644
--- a/src/box/errcode.h
+++ b/src/box/errcode.h
@@ -130,7 +130,7 @@ struct errcode_record {
 	_(ER_INDEX_TYPE, 13,			"Unsupported index type", "index", STRING, "space", STRING) \
 	_(ER_MODIFY_INDEX, 14,			"Cannot create or modify index", "index", STRING, "space", STRING, "details", STRING) \
 	_(ER_LAST_DROP, 15,			"Cannot drop the primary key in a system space", "space", STRING) \
-	_(ER_TUPLE_FORMAT_LIMIT, 16,		"Tuple format limit reached", "limit", UINT) \
+	_(ER_TUPLE_FORMAT_LIMIT, 16,		"Tuple format limit reached", "max", UINT) \
 	_(ER_DROP_PRIMARY_KEY, 17,		"Cannot drop primary key while secondary keys exist", "space", STRING) \
 	_(ER_KEY_PART_TYPE, 18,			"Supplied key field type does not match index part type", "fieldno", UINT, "expected", STRING) \
 	_(ER_EXACT_MATCH, 19,			"Invalid key part count in an exact match", "expected", UINT, "actual", UINT) \
@@ -170,7 +170,7 @@ struct errcode_record {
 	_(ER_BEFORE_REPLACE_RET, 53,		"Invalid return value of space:before_replace trigger: expected tuple or nil") \
 	_(ER_MULTISTATEMENT_TRANSACTION, 54,	"Can not perform in a multi-statement transaction", "action", STRING) \
 	_(ER_TRIGGER_EXISTS, 55,		"Trigger already exists", "trigger", STRING) \
-	_(ER_USER_MAX, 56,			"A limit on the total number of users has been reached", "limit", UINT) \
+	_(ER_USER_MAX, 56,			"A limit on the total number of users has been reached", "max", UINT) \
 	_(ER_NO_SUCH_ENGINE, 57,		"Space engine does not exist", "engine", STRING) \
 	_(ER_RELOAD_CFG, 58,			"Cannot set option dynamically", "option", STRING) \
 	_(ER_CFG, 59,				"Incorrect value for option", "option", STRING, "details", STRING) \
@@ -187,7 +187,7 @@ struct errcode_record {
 	_(ER_IDENTIFIER, 70,			"Invalid identifier", "id", STRING, "details", STRING) \
 	_(ER_DROP_FUNCTION, 71,			"Cannot drop function", "function_id", UINT, "details", STRING) \
 	_(ER_ITERATOR_TYPE, 72,			"Unknown iterator type", "type", STRING) \
-	_(ER_REPLICA_MAX, 73,			"Replica count limit reached", "limit", UINT) \
+	_(ER_REPLICA_MAX, 73,			"Replica count limit reached", "max", UINT) \
 	/* ER_INVALID_XLOG, 74, Unused */ \
 	/* ER_INVALID_XLOG_NAME, 75, Unused */ \
 	/* ER_INVALID_XLOG_ORDER, 76, Unused */ \
@@ -241,7 +241,7 @@ struct errcode_record {
 	_(ER_DECOMPRESSION, 124,		"Decompression error", "details", STRING) \
 	_(ER_INVALID_XLOG_TYPE, 125,		"Invalid xlog type", "expected", STRING, "actual", STRING) \
 	_(ER_ALREADY_RUNNING, 126,		"Failed to lock WAL directory and hot_standby mode is off", "directory", STRING) \
-	_(ER_INDEX_FIELD_COUNT_LIMIT, 127,	"Indexed field count limit reached", "limit", INT) \
+	_(ER_INDEX_FIELD_COUNT_LIMIT, 127,	"Indexed field count limit reached", "max", INT) \
 	_(ER_LOCAL_INSTANCE_ID_IS_READ_ONLY, 128, "The local instance is read-only", "instance_id", UINT) \
 	_(ER_BACKUP_IN_PROGRESS, 129,		"Backup is already in progress") \
 	_(ER_READ_VIEW_ABORTED, 130,		"The read view is aborted") \
@@ -272,7 +272,7 @@ struct errcode_record {
 	_(ER_NO_SUCH_GROUP, 155,		"Replication group does not exist", "group", STRING) \
 	/* ER_SQL_BIND_VALUE, 156, Unused */ \
 	_(ER_SQL_BIND_TYPE, 157,		"Bind value type is not supported", "value_type", STRING, "parameter", STRING) \
-	_(ER_SQL_BIND_PARAMETER_MAX, 158,	"SQL bind parameter limit reached", "limit", INT) \
+	_(ER_SQL_BIND_PARAMETER_MAX, 158,	"SQL bind parameter limit reached", "max", INT) \
 	_(ER_SQL_EXECUTE, 159,			"Failed to execute SQL statement", "details", STRING) \
 	_(ER_UPDATE_DECIMAL_OVERFLOW, 160,	"Decimal overflow", "operation", STRING, "field", STRING) \
 	_(ER_SQL_BIND_NOT_FOUND, 161,		"Parameter does not exist", "parameter", STRING) \
@@ -305,7 +305,7 @@ struct errcode_record {
 	_(ER_SQL_COLUMN_COUNT_MAX, 188,		"Too many columns", "space", STRING, "actual", INT, "max", INT) \
 	_(ER_HEX_LITERAL_MAX, 189,		"Hex literal exceeds the supported range", "literal", STRING) \
 	_(ER_INT_LITERAL_MAX, 190,		"Integer literal exceeds the supported range", "literal", STRING) \
-	_(ER_SQL_PARSER_LIMIT, 191,		"Parameter exceeds the limit", "parameter", STRING, "actual", INT, "limit", INT) \
+	_(ER_SQL_PARSER_LIMIT, 191,		"Parameter exceeds the limit", "parameter", STRING, "actual", INT, "max", INT) \
 	_(ER_INDEX_DEF_UNSUPPORTED, 192,	"Not supported in an index definition", "feature", STRING) \
 	/* ER_CK_DEF_UNSUPPORTED, 193, Unused */ \
 	_(ER_MULTIKEY_INDEX_MISMATCH, 194,	"Field is used as multikey in one index and as single key in another", "field", STRING) \
diff --git a/src/box/memtx_engine.cc b/src/box/memtx_engine.cc
index c1f730cf2..907e24e56 100644
--- a/src/box/memtx_engine.cc
+++ b/src/box/memtx_engine.cc
@@ -1818,7 +1818,8 @@ memtx_tuple_new_raw_impl(struct tuple_format *format, const char *data,
 		goto end;
 	});
 	if (unlikely(total > memtx->max_tuple_size)) {
-		diag_set(ClientError, ER_MEMTX_MAX_TUPLE_SIZE, total);
+		diag_set(ClientError, ER_MEMTX_MAX_TUPLE_SIZE, total,
+			 max_tuple_size);
 		error_log(diag_last_error(diag_get()));
 		goto end;
 	}

```

</details>

Reply:
Also changed `"actual"` to `"value"` where it used with `"min"` or `"max"`.

<details>
<summary>diff</summary>


```diff
diff --git a/src/box/errcode.h b/src/box/errcode.h
index a99dc7cba..5cc7a738e 100644
--- a/src/box/errcode.h
+++ b/src/box/errcode.h
@@ -145,7 +145,7 @@ struct errcode_record {
        _(ER_UNKNOWN_UPDATE_OP, 28,             "Unknown UPDATE operation", "argno", INT, "operation", STRING) \
        _(ER_UPDATE_FIELD, 29,                  "Field UPDATE error", "field", STRING, "details", STRING) \
        _(ER_FUNCTION_TX_ACTIVE, 30,            "Transaction is active at return from function") \
-       _(ER_KEY_PART_COUNT, 31,                "Key part count exceeds max allowed by the key definition", "max", UINT, "actual", UINT) \
+       _(ER_KEY_PART_COUNT, 31,                "Key part count exceeds max allowed by the key definition", "max", UINT, "value", UINT) \
        _(ER_PROC_LUA, 32,                      "Error in Lua function", "details", STRING) \
        _(ER_NO_SUCH_PROC, 33,                  "Function does not exist", "func", STRING) \
        _(ER_NO_SUCH_TRIGGER, 34,               "Trigger does not exist", "trigger", STRING) \
@@ -302,10 +302,10 @@ struct errcode_record {
        _(ER_SQL_UNKNOWN_TOKEN, 185,            "Unrecognized token", "line", INT, "column", INT, "token", STRING) \
        _(ER_SQL_PARSER_GENERIC, 186,           "Parse error", "details", STRING) \
        /* ER_SQL_ANALYZE_ARGUMENT, 187, Unused */ \
-       _(ER_SQL_COLUMN_COUNT_MAX, 188,         "Too many columns", "space", STRING, "actual", INT, "max", INT) \
+       _(ER_SQL_COLUMN_COUNT_MAX, 188,         "Too many columns", "space", STRING, "value", INT, "max", INT) \
        _(ER_HEX_LITERAL_MAX, 189,              "Hex literal exceeds the supported range", "literal", STRING) \
        _(ER_INT_LITERAL_MAX, 190,              "Integer literal exceeds the supported range", "literal", STRING) \
-       _(ER_SQL_PARSER_LIMIT, 191,             "Parameter exceeds the limit", "parameter", STRING, "actual", INT, "max", INT) \
+       _(ER_SQL_PARSER_LIMIT, 191,             "Parameter exceeds the limit", "parameter", STRING, "value", INT, "max", INT) \
        _(ER_INDEX_DEF_UNSUPPORTED, 192,        "Not supported in an index definition", "feature", STRING) \
        /* ER_CK_DEF_UNSUPPORTED, 193, Unused */ \
        _(ER_MULTIKEY_INDEX_MISMATCH, 194,      "Field is used as multikey in one index and as single key in another", "field", STRING) \
@@ -316,7 +316,7 @@ struct errcode_record {
        _(ER_FUNC_INDEX_FORMAT, 199,            "Key format does not match one defined in functional index", "index", STRING, "space", STRING, "details", STRING) \
        _(ER_FUNC_INDEX_PARTS, 200,             "Wrong functional index definition", "details", STRING) \
        _(ER_NO_SUCH_FIELD_NAME, 201,           "Field does not exist", "field", STRING) \
-       _(ER_FUNC_WRONG_ARG_COUNT, 202,         "Wrong number of arguments", "func", STRING, "actual", INT, "min", INT, "max", INT) \
+       _(ER_FUNC_WRONG_ARG_COUNT, 202,         "Wrong number of arguments", "func", STRING, "value", INT, "min", INT, "max", INT) \
        _(ER_BOOTSTRAP_READONLY, 203,           "Trying to bootstrap a local read-only instance as master") \
        _(ER_SQL_FUNC_WRONG_RET_COUNT, 204,     "SQL expects exactly one return value", "func", STRING, "actual", INT)\
        _(ER_FUNC_INVALID_RETURN_TYPE, 205,     "Invalid function return type", "func", STRING, "expected", STRING, "actual", STRING) \
```

</details>